server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailQueryMethod.scala

```diff
@@ -35,7 +35,7 @@ import org.reactivestreams.Publisher
 import play.api.libs.json.{JsError, JsSuccess}
 import reactor.core.scala.publisher.{SFlux, SMono}
 
-class EmailQueryMethod @Inject() (serializer: Serializer,
+class EmailQueryMethod @Inject() (serializer: EmailQuerySerializer,
```

Comment:
No need to inject this anymore, use `EmailQuerySerializer` directly line 68?

Reply:
The email query is still a variable.